### ISSUE TITLE
Creating a landing page for app.

### ISSUE THREAD

**Issue by anushbhatia** [smaranjitghose/DocLense#284]
Reference for it is here:https://www.resso.app/in

**Comment by Atharv-Joshi** [comment]
Can I work on this?

**Comment by anushbhatia** [comment]
Sure Atharv Joshi go ahead with it

**Comment by Atharv-Joshi** [comment]
Wouldn't a separate repo be efficient since the application has a lot of dependencies?

**Comment by anushbhatia** [comment]
Sure no problem I will update the base code for it.

**Comment by anushbhatia** [comment]
check out the docs repo in the current repo
https://github.com/smaranjitghose/DocLense/tree/docs

**Comment by Atharv-Joshi** [comment]
Hey! I thought I had to create it using flutter? I don't know react.

**Comment by anushbhatia** [comment]
Sure no problem we can take up the flutter web framework then

**Comment by Atharv-Joshi** [comment]
thanks!

**Comment by anushbhatia** [comment]
Updated the code have a look at it.

**Comment by Atharv-Joshi** [comment]
https://drive.google.com/drive/folders/1yQyUzp-xhzEF8kOt7joWDZQ96wM1YDGW?usp=sharing

can you check it out and give any suggestions?
I cannot figure out what the background should be

**Comment by anushbhatia** [comment]
Looks amazing go ahead to convert them from Figma/XD to flutter plugin. We will also have a section for contributors that with their GitHub profile as a carousel. You can later add them. Till then I will add these in the readme for references.

**Comment by Atharv-Joshi** [comment]
what do you mean by flutter plugin?
Like should I create a plugin for general landing page and then use it in this repo or directly code in this repo?
sorry for the trouble!

**Comment by anushbhatia** [comment]
No, actually there's a plugin that converts the design to flutter not that accurate but gives you the base code.
https://www.figma.com/community/plugin/844008530039534144

No need to design a plungin for this
